CHANGES.md

```diff
@@ -45,9 +45,12 @@ To be released.
  -  Added `BlockHeader.TotalDifficulty` property.  [[#666], [#917]]
  -  The existing `Pong` message type (with the type number `0x02`) was
     replaced by a new `Pong` message type
-    (with the type number `0x14`).  [[#459]. [#919]]
+    (with the type number `0x15`).  [[#459], [#919], [#920], [#930]]
  -  The `TimestampThreshold` between `Block<T>`s was changed from 15 minutes to
     15 seconds.  [[#922], [#925]]
+ -  Added new message types:
+     -  Added `GetChainStatus` message.  [[#920], [#930]]
+     -  Added `ChainStatus` message.  [[#920], [#930]]
```

Comment:
Do we need this? The interface does not seem to have changed.

Reply:
I've added these lines because this patch will lead to network inconsistency between swarms.